metrics/registry.go

```diff
@@ -1,19 +1,50 @@
 package metrics
 
+var Reg *Registry
+
+const (
+	defaultMetricPort int = 9999
+	COUNTER           int = 0
+	GAUGE             int = 1
+	SUMMARY           int = 2
+	UNTYPED           int = 3
+	HISTOGRAM         int = 4
+)
```

Comment:
let's define metrics which we would like to ingest from our app here, based on metric type we shall ingest it in the prometheus implementation

```suggestion
type MetricType int64

const (
	defaultMetricPort int = 9999
	Offset            MetricType = iota
)
```